services/app-web/src/tealium/tealium.ts

```diff
@@ -0,0 +1,320 @@
+import { User } from '../gen/gqlClient'
+import {
+    PageTitlesRecord,
+    RouteT,
+} from '../constants'
+import * as React from 'react';
+import {getRouteName} from '../routeHelpers';
+import {createScript} from '../hooks/useScript';
+import { getTealiumPageName } from './tealiumHelpers';
+import {recordJSException} from '../otelHelpers';
+
+// TYPES
+type TealiumDataObject = {
+    content_language: string
+    content_type: string
+    page_name: string
+    page_path: string
+    site_domain: 'cms.gov'
+    site_environment: string
+    site_section: string
+    logged_in: 'true' | 'false'
+    userId?: string // custom attribute
+    packageId?: string // custom attribute
+    tealium_event?: TealiumEvent // this is required by tealium, TBD what allowed values aer here, usually this is supposed to be configured first .
+}
+
+type ButtonEventStyle =
+    | 'default'
+    | 'primary'
+    | 'success'
+    | 'transparent'
```

Comment:
I think the only additional style we would need is `unstyled`. transparent is outline FYI. I can ask about this but for now please add `unstyled` to the types and then don't use any other types besides these events plus unstyled .